Makefile

```diff
@@ -65,6 +65,9 @@ endif
 ifeq ($(QUAI_STATS),true)
 	BASE_CMD += --quaistats ${STATS_NAME}:${STATS_PASS}@${STATS_HOST}
 endif
+ifeq ($(TRUSTED_STATS_PROVIDER),true)
+	BASE_CMD += --trustedstatsprovider
```

Comment:
please strip 'trusted' out of the terminology. go-quai is not trusting anyone.

As I understand it, there are two levels of stats reporting:
* partial reporting -- just append time and resource usage
* full reporting -- transaction & block detail stats in addition to partial ^

I think the flags should be named accordingly:
--stats_user
--stats_pass
--enable_stats //<--- only sends the 'non-trusted stats'
--send_full_stats // <--- sends the so called 'trusted stats'

Reply:
Fixed